pgrollup/parser_pg.py

```diff
@@ -0,0 +1,290 @@
+import sys
+from lark import Lark, Transformer
+
+def parse_create(sql):
+    '''
+    Example expected output:
```

Comment:
This must use doctests.  What you have cannot be checked automatically by python, but doctests can.

Reply:
Currently, there's no tests for any of the code that you've written here, so there's no way for me to know if it is actually working or not.  Good tests are the most important part of any pull request.